js/shared/pollimporter.js

```diff
@@ -88,6 +88,7 @@ export default class PollImporter {
         const importCfg = JSON.parse(json);
         $this.projectTransform = TransformFactory.create(importCfg);
       } catch (err) {
+        // eslint-disable-next-line no-console
```

Comment:
I don't think we should disable the `no-console` warnings. It is a good reminder on where `console` statements are being used so that we can explore removing them in the future depending on the direction the importer takes.

Reply:
So only ignore if we plan to leave them in?
I find extra warnings can make it harder to find warnings we actually don't want to check in.

Reply:
Why should we plan to leave any in? Are we expecitng these messages to be useful to end users? If that is the case should there be a UI instead for such messages rather than the dev console?

Reply:
I'm not sure.  They are developer's using it right?  So used to the console?  But, ya, a built in error console would be more useful for failure to connect error messages, unexpected exceptions, etc.